apps/studio/src/features/editing-experience/components/CreateCollectionPageModal/DetailsScreen.tsx

```diff
@@ -142,7 +142,10 @@ export const CreateCollectionPageDetailsScreen = () => {
               </FormControl>
 
               {/* Section 2: Page URL */}
-              <FormControl isInvalid={!!errors.permalink}>
+              <FormControl
+                isInvalid={!!errors.permalink}
+                display={type === "link" ? "None" : "auto"}
```

Comment:
nit: should be lowercase?

```suggestion
                display={type === "link" ? "none" : "auto"}
```